src/vsr/journal.zig

```diff
@@ -207,6 +233,7 @@ pub fn Journal(comptime Replica: type, comptime Storage: type) type {
 
             self.dirty.deinit(allocator);
             self.faulty.deinit(allocator);
+            allocator.free(self.prepare_checksums);
             allocator.free(self.headers);
             allocator.free(self.headers_iops);
```

Comment:
```suggestion
            allocator.free(self.headers);
            allocator.free(self.headers_iops);
            allocator.free(self.prepare_checksums);
```

To match the order on disk.